src/cdk/scripts/manage-exports.py

```diff
@@ -0,0 +1,1279 @@
+        internal_prefixes = ["AWS::", "CDK::", "cdk-", "CdkBootstrap", "StagingBucket"]
+        return any(export_name.startswith(prefix) for prefix in internal_prefixes)
+
+    def _categorize_export(self, export_name: str) -> str:
+        """Categorize export based on prefix and naming patterns."""
+        # First check for prefix-based categorization
+        if ":" in export_name:
+            prefix = export_name.split(":", 1)[0]
+            if prefix == "public":
+                # For public exports, use detailed categorization
+                name_lower = export_name.lower()
+
+                if any(
+                    term in name_lower for term in ["vpc", "subnet", "cidr", "network"]
+                ):
+                    return "Networking"
+                elif any(
+                    term in name_lower
+                    for term in ["database", "db", "rds", "aurora", "dynamo"]
+                ):
+                    return "Database"
+                elif any(term in name_lower for term in ["bucket", "s3", "assets"]):
+                    return "Storage"
+                elif any(
+                    term in name_lower for term in ["cluster", "ecs", "eks", "compute"]
+                ):
+                    return "Compute"
+                elif any(
+                    term in name_lower for term in ["api", "gateway", "endpoint", "url"]
+                ):
+                    return "API"
+                elif any(
+                    term in name_lower for term in ["security", "role", "policy", "waf"]
+                ):
+                    return "Security"
+                elif any(
+                    term in name_lower
+                    for term in ["monitor", "log", "metric", "opensearch"]
+                ):
+                    return "Observability"
+                else:
+                    return "Other"
+            elif prefix == "private":
+                return "Private"
+            else:
+                # Unknown prefix, treat as other
+                return "Other"
+        else:
+            # No prefix (no colon), categorize as internal-cdk
+            return "internal-cdk"
+
+    def _clean_export_name_for_display(self, export_name: str) -> str:
+        """
+        Clean export name for dashboard display by removing internal prefixes.
+
+        The 'public:' and 'private:' prefixes are used internally for filtering
+        but should not be shown to users in the dashboard.
+        """
+        # Remove common prefixes used for categorization
+        prefixes_to_remove = ["public:", "private:"]
+
+        for prefix in prefixes_to_remove:
+            if export_name.startswith(prefix):
+                return export_name[len(prefix) :]  # noqa: E203
+
+        return export_name
+
+    def _is_url_value(self, value: str) -> bool:
+        """Check if export value appears to be a URL."""
+        return value.startswith(("http://", "https://"))
+
+    def _get_console_url(
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this function is 17. By comparison, 98% of the functions in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the function has a high number of decisions and it can make the logic difficult to understand and test.

We recommend that you simplify this function or break it into multiple functions. For example, consider extracting the code block on lines 661-666 into a separate function.